browser/iframe-browser.js

```diff
@@ -0,0 +1,106 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager      = brackets.getModule("command/CommandManager"),
+        Menus               = brackets.getModule("command/Menus"),
```

Comment:
I'm talking about line 5--my comment is right under that line, so refers to the preceeding line: ``` Menus               = brackets.getModule("command/Menus"),```  You don't use `Menus` anywhere, so you can remove that entire line.